creator-node/src/app.ts

```diff
@@ -19,18 +19,23 @@ const {
   URSMRequestForSignatureReqLimiter,
   batchCidsExistReqLimiter,
   getRateLimiterMiddleware
-} = require('./reqLimiter')
-const config = require('./config')
-const {
-  exponentialBucketsRange
-} = require('./services/prometheusMonitoring/prometheusUtils')
-const healthCheckRoutes = require('./components/healthCheck/healthCheckController')
-const contentBlacklistRoutes = require('./components/contentBlacklist/contentBlacklistController')
-const replicaSetRoutes = require('./components/replicaSet/replicaSetController')
-
-function errorHandler(err, req, res, next) {
-  req.logger.error('Internal server error')
-  req.logger.error(err.stack)
+} from './reqLimiter'
+import config from './config'
+import { exponentialBucketsRange } from './services/prometheusMonitoring/prometheusUtils'
+import healthCheckRoutes from './components/healthCheck/healthCheckController'
+import contentBlacklistRoutes from './components/contentBlacklist/contentBlacklistController'
+import replicaSetRoutes from './components/replicaSet/replicaSetController'
+
+type RequestWithLogger = Request & { logger: Logger }
+
+function errorHandler(
+  err: any,
+  req: Request,
```

Comment:
why can't this be RequestWithLogger? the spam of casting the req every time it's used after is a little distracting

Reply:
That seems unruly

Reply:
Lemme play around with it